### DISCUSSION TITLE
판매(ZEM)가 0인 이슈 문의

### DISCUSSION THREAD

**Discussion by BaePO** [naverz/zepeto-studio-kor#1013] [5. Q&A]
안녕하세요. 월드 콘텐츠 판매를 확인하던 도중에 이상한 점이 있어 문의드립니다.

7월 12일, 13일에 각각 6개, 9개의 콘텐츠가 판매됐습니다. 
하지만 첨부해드리는 스크린샷을 보시면 판매 (ZEM)은 모두 0으로 나와있습니다. 

월드 내에서 IWP가 정상적으로 작동하지 않나 싶어 직접 젬을 사용하여 구매를 해봤지만 젬 차감과 구매완료 시스템에는 이상이 없었습니다.

혹시 판매 정보를 알 수 있는 방법이나 관련된 버그가 있는 지 궁금합니다. 감사합니다! (IWP는 1.0.3 버전을 사용하고 있습니다.)
![KakaoTalk_20220714_113209441](https://user-images.githubusercontent.com/96255741/178889367-c409b4da-b60b-494c-a03a-75276222fcd2.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이 부분은 제페토 고객센터에 문의해주시면 도움을 받을 수 있을 것 같습니다. 
https://support.zepeto.me/hc/ko

감사합니다.

**Reply by BaePO** [reply]
감사합니다!
